micro-rdk/src/common/grpc.rs

```diff
@@ -1474,17 +1542,14 @@ impl<R> Service<Request<body::Incoming>> for GrpcServer<R>
 where
     R: GrpcResponse + Body + Clone + 'static,
 {
-    type Response = Response<R>;
+    type Response = Response<BoxBody<Bytes, GrpcError>>;
```

Comment:
Note that `R` no longer appears here, and we aren't using the `GrpcResponse` stuff at all. For that matter, we aren't using the `GrpcBody` trait either. I left all of that alone in the interest of getting this up for review, since removing that stuff would require compensating changes in the WebRTC based GrpcServer infra. A follow-up will refactor this, and hopefully simplify it all.